extensions/amp-autocomplete/0.1/amp-autocomplete.js

```diff
@@ -323,16 +323,22 @@ export class AmpAutocomplete extends AMP.BaseElement {
   /**
    * Create and return <div> element from given plan-text item.
    * @param {string} item
+   * @param {string=} substring
    * @return {!Element}
    * @private
    */
-  createElementFromItem_(item) {
+  createElementFromItem_(item, substring) {
```

Comment:
nit: prefix optional parameters with `opt_`.

Reply:
Prefixing with `opt_` is no longer required, but the internal styleguide asks for a default value, ie `substring = ''` or `null` or `undefined`
http://go/js-style#features-functions-default-parameters

Reply:
Thanks, TIL!